### ISSUE TITLE
[Bug] Disabling items and blocks breaks tags

### ISSUE THREAD

**Issue by MJRamon** [paulevsGitch/BetterNether#474]
### What happened?

Disabling everything related to Cincinnasite (items and blocks) break Minecraft tags, thus breaking datapack functionality.

```
[04:43:20] [Worker-Main-17/ERROR]: Couldn't load tag minecraft:wooden_doors as it is missing following references: betternether:bone_cincinnasite_door (from Better Nether)
[04:43:20] [Worker-Main-17/ERROR]: Couldn't load tag minecraft:anvil as it is missing following references: betternether:cincinnasite_anvil (from Better Nether)
[04:43:20] [Worker-Main-17/ERROR]: Couldn't load tag minecraft:walls as it is missing following references: betternether:cincinnasite_wall (from Better Nether)
```

### BetterNether

better-nether-6.0.6

### BCLib

bclib-1.1.2

### Fabric API

fabric-api-0.44.0+1.18

### Fabric Loader

0.12.12

### Minecraft

1.18.1

**Comment by quiqueck** [comment]
This is part of ongoing refactoring. We work on it, but it will take some time :)

**Comment by quiqueck** [comment]
Some parts of the fix will be in 6.0.7.

**Comment by quiqueck** [comment]
This should be fixed in 6.0.8. Please re-open or create a new issue if the problem persists.
